src/SprykerSdk/Shared/Benchmark/Command/AbstractCommandBuilder.php

```diff
@@ -0,0 +1,174 @@
+    }
+
+    /**
+     * @return string
+     */
+    protected function getFallbackBootstrapFolder(): string
+    {
+        $moduleRootFolder = dirname(__DIR__, 6);
+
+        return realpath(sprintf('%s/%s/%s', $moduleRootFolder, 'bootstrap', $this->getApplication()));
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer
+     *
+     * @return string
+     */
+    protected function getTestDirectory(PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer): string
+    {
+        return $phpBenchConfigurationTransfer->getTestDirectory() ?: $this->getApplicationTestsDirectory();
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer
+     *
+     * @return string
+     */
+    protected function getBootstrapPath(PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer): string
+    {
+        $bootstrapPath = $this->getPathToBootstrap($phpBenchConfigurationTransfer);
+
+        return static::BENCHMARK_CLI_BOOTSTRAP_CONFIG . $bootstrapPath;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer
+     *
+     * @return string
+     */
+    protected function getIterations(PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer): string
+    {
+        $iterations = $phpBenchConfigurationTransfer->getIterations() ?: $this->getIterationConfig();
+
+        return static::BENCHMARK_CLI_ITERATIONS_CONFIG . $iterations;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer
+     *
+     * @return string
+     */
+    protected function getRevolutions(PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer): string
+    {
+        $revolutions = $phpBenchConfigurationTransfer->getRevolutions() ?: $this->getRevolutionConfig();
+
+        return static::BENCHMARK_CLI_REVOLUTIONS_CONFIG . $revolutions;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer
+     *
+     * @return string
+     */
+    protected function getReport(PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer)
+    {
+        $report = $phpBenchConfigurationTransfer->getReport() ?: $this->getReportConfig();
+
+        return static::BENCHMARK_CLI_REPORT_CONFIG . $report;
+    }
+
+    /**
+     * @return string
+     */
+    protected function getTimeUnit(): string
+    {
+        return static::BENCHMARK_CLI_TIME_UNIT_CONFIG . 'milliseconds';
+    }
+
+    /**
+     * @return string
+     */
+    abstract protected function getApplication(): string;
+
+    /**
+     * @return string
+     */
+    abstract protected function getApplicationTestsDirectory(): string;
+
+    /**
+     * @return int
+     */
+    abstract protected function getIterationConfig(): int;
```

Comment:
IMO it doesn't make sense for these abstract methods per each application layer.
If I got it correctly we can set the default value inside the console command for Yves, Zed, and Glue.